tests/integration_tests/test_botorch.py

```diff
@@ -410,3 +410,22 @@ def test_call_after_trial_of_independent_sampler() -> None:
     ) as mock_object:
         study.optimize(lambda _: 1.0, n_trials=1)
         assert mock_object.call_count == 1
+
+
+@pytest.mark.parametrize("device", [None, torch.device("cpu"), torch.device("cuda:0")])
```

Comment:
The test of `device = torch.device("cuda:0")` seems to be skipped in CI now.
From https://github.com/optuna/optuna/actions/runs/3344000982/jobs/5537846173, 
```
tests/integration_tests/test_botorch.py .....................s
```

How about adding a TODO comment for future work to enable the GPU test on CI?

Reply:
Yes, you are right. I created a skip case intendedly to support both GPU-available and GPU-unavailable environments because we cannot assume all developers have GPUs on their local machines to run the test locally. 
If we create such TODO, GitHub issue is supposed to be more appropriate than in this test function.